vendor-extra/ContentPageBundle/src/Event/CreateContentPageEvent.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentPageBundle\Event;
+
+use FluentDOM\DOM\Document;
+use Symfony\Component\EventDispatcher\Event;
+
+class CreateContentPageEvent extends Event
+{
+    public const NAME = 'libero.page.content';
+
+    private $content = [];
+    private $context;
```

Comment:
A lot of these things could be extracted into a more generic page event (eg search page, listing page).

Reply:
(when we have >=2 pages)